dluhc-component-library/src/components/siteSelectionForm/SiteSelectionForm.tsx

```diff
@@ -11,6 +11,27 @@ interface SiteSelectionForm {
   data?: SiteSelectionFormSchema;
 }
 
+const createValidationSchema = (
+  key: string,
+  formSchema: SiteSelectionFormSchema,
+) => {
+  let validationShape: any;
```

Comment:
Would limiting this to the types of schema like `StringSchema` and `NumberSchema` work, or does it need to be any?

Reply:
Correct me if I'm wrong @amjones-scottlogic  but I think I'll be doing that in the next PR where I add validation for the rest of the question types (number/multiselect etc)

Reply:
Nope, I missed coming back to this! It should be done in this PR

Reply:
Nope, I missed coming back to this! We should fix it in this PR